src/app/components/LiveLabel/index.stories.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { withKnobs } from '@storybook/addon-knobs';
+import { withServicesKnob } from '../../legacy/psammead/psammead-storybook-helpers/src';
+import LiveLabel from '../../legacy/psammead/psammead-live-label/src/index';
+import md from '../../legacy/psammead/psammead-live-label/README.md';
+import { StoryProps } from '../../models/types/storybook';
+import services from '../../../server/utilities/serviceConfigs';
+import Heading from '../Heading';
+import InlineLink from '../InlineLink';
+
+interface Props extends StoryProps {
+  dir: string;
```

Comment:
We have a `Direction` type that could be used here, but it can be added in the migration of the LiveLabel component, so all good!